skatepark/functions.php

```diff
@@ -40,5 +40,13 @@ function skatepark_support() {
 function skatepark_scripts() {
 	// Enqueue front-end styles.
 	wp_enqueue_style( 'skatepark-styles', get_stylesheet_directory_uri() . '/assets/theme.css', array( 'blockbase-ponyfill' ), wp_get_theme()->get( 'Version' ) );
+
+	// Allow the responsive menu label to be translated.
+	$menu_label_css = '
```

Comment:
This feels very messy to me, which makes me think it should go in the block.

In the meantime, maybe it should live in blockbase.

Reply:
but blockbase doesn't have the Menu text. I think this is probably [planned for the navigation block](https://github.com/WordPress/gutenberg/issues/22824#issuecomment-789584886)

Reply:
Ok, I'm going to remove it from the theme for now and open an issue for it in Gutenberg.